src/popup/components/JobDetails.ts

```
@@ -220,15 +305,38 @@
+    vm.questions = questions
+      .map((q) => (typeof q === 'object' && q !== null && 'question' in q && typeof (q as { question?: unknown }).question === 'string'
+        ? (q as { question: string }).question
+        : null))
+      .filter((q): q is string => q !== null);
 
-    const jobDescription = details?.opening?.job?.description;
-    if (jobDescription && jobDescription.trim().length > 0) {
+    // description may be missing/null in API; optional access is intentional
+    // _Opening kept for docs readability while avoiding no-unused-vars
+    type _Opening = { job?: { description?: string } }; // docs: API shape reference (unused by code)
+    const openingObj = (details && typeof details === 'object') ? (details as Record<string, unknown>).opening as Record<string, unknown> | null | undefined : null;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'openingObj'. Consider to delete it or rename it.

'openingObj' is defined here:


(lint/suspicious/noRedeclare)